client/Components/GameBoard/PlayerStats.jsx

```diff
@@ -56,6 +57,16 @@ export class PlayerStats extends React.Component {
         );
     }
 
+    writeChatToClipboard(event) {
+        event.preventDefault();
+        let messagePanel = document.getElementsByClassName('messages panel')[0];
```

Comment:
This will work, but it’s not very “react”. It should use refs to get the handle to the Dom element